src/cares_wrap.cc

```diff
@@ -2052,6 +2066,11 @@ void SetServers(const FunctionCallbackInfo<Value>& args) {
   ChannelWrap* channel;
   ASSIGN_OR_RETURN_UNWRAP(&channel, args.Holder());
 
+  if (channel->activity_query_count()) {
+    env->ThrowError("Activity query(s) is/are open.");
```

Comment:
`Activity` → `Active`? `querys` is not the plural of `query`. Maybe just use something simple like `There are pending queries.` ?